packages/@aws-cdk/aws-iot-actions/test/stepfunctions/integ.stepfunctions-action.ts

```diff
@@ -0,0 +1,25 @@
+import * as iot from '@aws-cdk/aws-iot';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as actions from '../../lib';
```

Comment:
Oh, I missed this before, but as a new test, this needs to use the `IntegTest` construct and then run it with the `--update-on-failed` flag.